### PR TITLE
Fix misplacement of functions

### PR DESCRIPTION
**Pull request by marcorosa**
There is no  reason to place  `generate_new_key_pair`  function in `__init__.py` file.
It also raises an exception if someone imports `http.Client` class.

### CODE REVIEW

**Comment by marcorosa** [comment]
I agree with you. Changed code accordingly

**Comment by marcorosa** [comment]
I'm sorry, I closed the pull request by mistake. I think we are ready to merge

**Comment by steenzout** [comment]
@marcorosa please make the PR to the `develop` branch.

**Comment by marcorosa** [comment]
ok. done

File: storj/http.py [hunk 1]
```diff
@@ -66,7 +66,7 @@ def authenticate(self, ecdsa_private_key=None):
         if isinstance(ecdsa_private_key, SigningKey):
             self.private_key = ecdsa_private_key
             self.public_key = self.private_key.get_verifying_key()
-            self.public_key_hex = ecdsa_to_hex(self.public_key)
+            self.public_key_hex = ecdsa_to_hex(self.public_key.to_string())
```
Comment (steenzout) [inline storj/http.py]: `to_string()` does not exist.
it was `str(self.public_key)`.
fixing on #89 PR.

Comment (steenzout) [inline storj/http.py]: @marcorosa ^

Comment (marcorosa) [inline storj/http.py]: are you sure? I tested it in Python2 and it worked.
Anyway, `public_key.to_string()` and `str(public_key)` doesn't change, so also the `str` notation is ok

Comment (steenzout) [inline storj/http.py]: it broke on one of the unit tests.

**Pull request merged**